jenkins/automation/kubernetes/cortx-all-image-tagging.groovy

```diff
@@ -69,7 +69,7 @@ pipeline {
         always {
             cleanWs()
             script {
-                env.docker_image_location = "https://github.com/Seagate/cortx-re/pkgs/container/cortx-all"
+                env.docker_image_location = "${BASE_IMAGE_NAME}"
```

Comment:
This needs to be a GitHub URL instead of the only Image name.

Reply:
Done